lib/services/queue_service.dart

```diff
@@ -448,7 +448,9 @@ class QueueService {
           initialIndex: items["current"]!.isNotEmpty || items["queue"]!.isNotEmpty ? items["previous"]!.length : 0,
           // Always restore queues as linear to prevent them being reshuffled while restoring
           order: FinampPlaybackOrder.linear,
-          beginPlaying: isReload && (_audioHandler.playbackState.valueOrNull?.playing ?? false),
+          beginPlaying:
+              (isReload && (_audioHandler.playbackState.valueOrNull?.playing ?? false)) ||
+              (FinampSettingsHelper.finampSettings.autoplayRestoredQueue && droppedTracks == 0),
```

Comment:
```suggestion
              isReload ? (_audioHandler.playbackState.valueOrNull?.playing ?? false) :
              (FinampSettingsHelper.finampSettings.autoplayRestoredQueue && droppedTracks == 0),
```
This ensures queues don't suddenly start playing when they're paused, network changes, and the queue is either manually or automatically reloaded.